### PR TITLE
unify application entrypoint

### PR DESCRIPTION
**Pull request by denisbedin**
https://docs.microej.com/en/feature-sdk6_unify_application_entrypoint/

### CODE REVIEW

File: SDK6UserGuide/createProject.rst [hunk 1]
```diff
@@ -267,64 +267,54 @@ Application Project
...
-      microej {
-        applicationMainClass = "com.mycompany.Main"
-      }
+   version=1.0.0
+   name=MY-KERNEL
 
-  - Create a file named ``kernel.kf`` in the ``src/main/resources`` folder.
-    This file is a property file which must contain at least the ``version`` property, 
-    and optionally the ``name`` property (defaults to ``KERNEL``), for example::
+- If your Application requires advanced features, for example the `FeatureEntryPoint.stop()`_ method:
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/createProject.rst:284]: Shouldn't we move this chapter in a dedicated page or at least a dedicated section, less visible, to keep the default flow lighter?

File: SDK6UserGuide/moduleNatures.rst [hunk 2]
```diff
@@ -475,6 +476,31 @@ This task is used by the following module natures:
 
 - :ref:`sdk6_module_natures.application`
 
+.. _sdk6_module_natures.tasks.wrapApplication:
+
+wrapApplication
+^^^^^^^^^^^^^^^
+
+**Description**: Wraps the Application.
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/moduleNatures.rst:484]: Should be more detailed IMO: "Wraps the Application class to be able to run ..."

File: SDK6UserGuide/sdk6MigrationNotes.rst [hunk 3]
```diff
@@ -10,6 +10,51 @@ From 0.14.0 to 0.15.0
 This section applies if MicroEJ SDK 6 ``0.15.0`` is used on a project that was created using MicroEJ SDK 6 ``0.14.0`` 
 or lower.
 
+Unification of Application EntryPoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (tdelhomenie-microej) [inline SDK6UserGuide/sdk6MigrationNotes.rst:14]: This chapter should go in another section "from 0.15.0 to 0.16.0".

**Review by tdelhomenie-microej — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/sdk6MigrationNotes.rst (hunk 3 above)
Comment (denisbedin) [reply SDK6UserGuide/sdk6MigrationNotes.rst:14]: Done.

File: SDK6UserGuide/moduleNatures.rst (hunk 2 above)
Comment (denisbedin) [reply SDK6UserGuide/moduleNatures.rst]: Done.

File: SDK6UserGuide/createProject.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/createProject.rst]: I created a new HowTo Guide : https://docs.microej.com/en/feature-sdk6_unify_application_entrypoint/SDK6UserGuide/howtos/createFeatureClass.html

**Review by tdelhomenie-microej — APPROVED** [review]

File: SDK6UserGuide/createProject.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/createProject.rst]: I changed the title

**Pull request merged**